src/dbus_api/macros.rs

```diff
@@ -84,3 +84,14 @@ macro_rules! properties_footer {
         }
     };
 }
+
+macro_rules! pre_pool_op {
+    ($oper:tt, $arg:ident; $postoper:tt $(, $args:ident)*; $engine_op:expr) => {{
+        info!($oper, $arg);
+        let result = $engine_op;
+        if let Ok(ref test_uuid) = result {
```

Comment:
You named it `test_uuid` while testing, but we probably should change the name to something without "testing" before merging this into our codebase.

Reply:
Resolved; changed to "uuid".